.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### What's being changed:
+
+
+### Review checklist
+
+- [ ] Documentation has been updated, if necessary. Link to changed documentation:
+- [ ] Chaos pipeline run or not necessary. Link to pipeline:
```

Comment:
I disagree on this, we had many situations where chaos pipeline actually revealed a bug in the PR just before it went to the master branch. Running chaos pipeline for PR's (not all, just only for those that touch our core features) should be mandatory as in case of potential bug it's obvious that this change introduced this bug, but when merged into master makes it then difficult to pin point which PR introduced it.